functions/src/api/index.ts

```diff
@@ -24,6 +26,12 @@ app.use('/api/v1', routerV1)
 app.use('/api/v1-private', routerPrivate)
 app.use('/api/v1-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec))
 
+if (isUsingEmulator()) {
+    app.get('/api/send-topic-notification', (req, res) =>
```

Comment:
le truc salace :D 
je l'enlèverai quand je ferais l'api pour les notif

Reply:
why ? It allows to have an easy way to test notifications without bothering production code

Reply:
oué, je changerai plus tard :p